wp-includes/rest-api/endpoints/class-wp-rest-customize-partials-controller.php

```diff
@@ -0,0 +1,327 @@
+<?php
+/**
+ * REST API: WP_REST_Customize_Partials_Controller class
+ *
+ * @package WordPress
+ * @subpackage REST_API
+ * @since ?.?.?
+ */
+
+/**
+ * Core class to access partials via the REST API.
+ *
+ * @since ?.?.?
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Customize_Partials_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 4.7.0
+	 * @access public
+	 */
+	public function __construct() {
+		$this->namespace = 'customize/v1';
+		$this->rest_base = 'partials';
+	}
+
+	/**
+	 * Ensure customize manager.
+	 *
+	 * @return WP_Customize_Manager Manager.
+	 * @global WP_Customize_Manager $wp_customize
+	 */
+	public function ensure_customize_manager() {
```

Comment:
I suppose this method is preset in each of the customize controller classes? Perhaps there should be an `abstract` parent class like `WP_REST_Customize_Controller` that defines this method?